v2/internal/pkg/cli/const.go

```diff
@@ -18,7 +18,7 @@ const (
 	operatorImageExtractDir       string = "hold-operator"
 	operatorCatalogsDir           string = "operator-catalogs"
 	signaturesDir                 string = "signatures"
-	registryLogFilename           string = "registry.log"
+	registryLogFilename           string = "registry-%s.log"
```

Comment:
Since this is not constant anymore, do we still needs this variable? Why is `oc-mirror-%s.log` not a variable but this is?